### PR TITLE
[Resource Action] LPS-117531

### PR DESCRIPTION
**Pull request by dantewang**

### CODE REVIEW

**Comment by dantewang** [comment]
ci:test:sf

**Comment by dantewang** [comment]
ci:test

**Comment by dantewang** [comment]
ci:test:relevant

**Comment by tinatian** [comment]
@dantewang @yuhai 
This pull is too big to review, please try to break it down into small pulls, or squash commits to show the logic clearly.

For example, these two commits seem to be unrelated to this pull, and should be able to commit separately.
LPS-117531 Remove service.configurator.portlet.ids, this should have been deprecated since LPS-68841
LPS-117531 Remove "backwards compatibility" of service.configurator.portlet.ids" by LPS-68841, portlet resources are always read/created and checked upon portlet deployment

File: portal-impl/src/com/liferay/portal/security/permission/ResourceActionsImpl.java [hunk 1]
```diff
@@ -996,7 +1003,8 @@ private String _normalizePortletName(
 
 	private void _read(
 			Portlet portlet, String servletContextName, ClassLoader classLoader,
-			String source, Set<String> portletNames)
+			String source, Set<String> resourceNames,
+			boolean readPortletResource)
```
Comment (tinatian) [inline portal-impl/src/com/liferay/portal/security/permission/ResourceActionsImpl.java]: @dantewang @yuhai 
I do not like the new add parameter `readPortletResource`, could you please try to find a better way to do this?

Thanks.
Tina.

**Pull request closed without merging**